man/man8/swtpm.pod

```diff
@@ -317,6 +323,10 @@ The I<--key> option supports the I<fd=> parameter.
 
 The I<--key> option supports the I<pwdfd=> parameter.
 
+=item B<nvram-backend-file>
+
+The I<--tpmstate> option supports the I<backend=file> option.
```

Comment:
... supports the I<backend=file> and I<backend_uri=uri> parameters.

I hope that everything that's needed by the backends can be encoded into the backend_uri parameter...

Reply:
Yes, we can encode parameters in uri for example `backend_uri=iscsi://...,key1=val1,key2=val2`.